src/test/java/HomeWork24.java

```diff
@@ -0,0 +1,17 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.LoginPage;
+
+import java.net.MalformedURLException;
+
+public class HomeWork24 extends BaseTest2{
+    public String url = "https://qa.koel.app/";
+    @Test
+    public void launchGrid() throws MalformedURLException {
+        loginGrid();
+    }
+    public void loginGrid() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.loginMainPage();
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/crpro108/A55-QA-Automation-Framework/assets/104761482/2501b44f-82e7-45bf-ba25-72cefc149902)